composer.json

```diff
@@ -13,7 +13,8 @@
     "require": {
         "php": ">=5.4",
         "php-http/httplug": "^1.0.0-alpha2@dev",
-        "php-http/message-factory": "^0.3"
+        "php-http/message-factory": "^0.3",
+        "php-http/client-tools": "^0.1@dev"
```

Comment:
The EmulateAsyncClient uses two traits from the Tools repo. Also, the promises are moved to the tools repo as well, which are used in the same class.